backend/inspirehep/legacy/views.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import flask
+from flask import Blueprint, abort, request
+from sqlalchemy.orm.exc import NoResultFound
+
+from inspirehep.pidstore.api import PidStoreBase
+from inspirehep.records.utils import get_pid_for_pid
+
+blueprint = Blueprint("inspirehep_legacy", __name__, url_prefix="")
+
+
+@blueprint.route("/record/<path:recid>", methods=("GET",))
+def redirect_record(recid):
+    try:
+        endpoint = PidStoreBase.get_endpoint_from_recid(recid)
+    except NoResultFound:
+        abort(404)
+
+    return flask.redirect(f"/{endpoint}/{recid}", 301)
+
+
+@blueprint.route("/author/profile/<path:bai>", methods=("GET",))
+def redirect_author(bai):
+    recid = get_pid_for_pid("bai", bai, "recid")
+    if not recid:
+        abort(404)
+
+    return flask.redirect(f"/authors/{recid}", 301)
+
+
+@blueprint.route("/search", methods=("GET",))
+def redirect_query():
+    collection_equivalence = {
+        "HEP": "literature",
+        "HepNames": "authors",
+        "Conferences": "conferences",
```

Comment:
conferences don't need anymore, we have them on prod now

Reply:
But we still want to redirect old links right?

Reply:
we didn't do it for Jobs, that's why I think Conferences should follow the same behaviour. WDYT @michamos ?

Reply:
I don't think there is any harm in redirecting everything, even if it's probably most useful for HEP.